src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/PackageSourceValidator.cs

```diff
@@ -13,11 +13,18 @@ namespace NuGet.PackageManagement.VisualStudio.Options
     internal static class PackageSourceValidator
     {
         internal static PackageSource FindExistingOrCreate(
+            string packageSourceId,
```

Comment:
now that I read the code a few times, isn't this effective "oldName" and then the name string is "newName" (which the same as old in most scenarios).

I feel like the name could be better. 

We're calling it packagesourceid, but when looking up in other methods we are comparing against the name, so I think "name" should be part of the name. 

originalName?
currentName?

Reply:
It's called ID because it's the unique identifier of the source item. It happens to be the Name of the source. It may be confusing to see "Name" and something like "foo_Name" and understand that the "foo_Name" is specifically the ID.

Reply:
> has the same implementation as FindByName too.

There is no more FindByName. This PR renames it. I've reached out to speak offline when you're available.

Reply:
Yup, we chatted offline, resolving this.